src/api/controllers/articlesController.js

```diff
@@ -0,0 +1,63 @@
+import model from '../models/index';
+import status from '../../helpers/constants/status.codes';
+/**
+ * @class
+ */
+export default class Articles {
+  /**
+ * This function gets all the articles that are stored in the article table
+ *
+ * @static
+ * @param {*} req the request
+ * @param {*} res the response to be sent
+ * @memberof Articles
+ * @returns {Object} res
+ */
+  static async getArticles(req, res) {
+    const { Article } = model;
+    const articles = await Article.findAll();
+    const ZERO = 0;
+
+    if (articles.length > ZERO) {
+      res.status(status.OK).json({
+        status: 200,
```

Comment:
@truestbyheart Instead of hardcoded values like 200 and 404 etc, let's use constants like `status.OK` and `status.NOT_FOUND` which have already been defined.